package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && mocha --timeout 20000"
+    "test": "if-node-version \">=4\" xo && mocha --timeout 20000"
```

Comment:
@SBoudrias I just checked the Travis CI output. `mocha --timeout 20000` is definitely still be executed in Node.js 0.10 and 0.12:
https://travis-ci.org/yeoman/update-notifier/jobs/169133269